Claudia/src/main/java/com/endava/claudia/task/four/User.java

```diff
@@ -0,0 +1,16 @@
+package com.endava.claudia.task.four;
+
+public class User extends AbstractUser {
+
+    public User(String username, String addressType, Address address) {
+
+        super(username);
```

Comment:
Please see the coment from AdminUser class constructor!

Reply:
Done